### ISSUE TITLE
Handle new/empty direct message narrows better

### ISSUE THREAD

**Issue by Niloth-p** [zulip/zulip-terminal#1480]
On opening DM narrows with a new user (someone you haven't previously had a private conversation with),
- The message narrow displayed in the recipient bar, and the conversation focus displayed in the message search box are both not updated.
Related issues:
#1343
#607 

- If you decide to leave that narrow (without sending any messages), you are unable to. You need to open some other narrow manually. i.e the Esc key does not go back. Nor does the 'a' hotkey take you to all messsages.
One needs to use any of the other narrow hotkeys or navigate to a topic/stream/user and open that narrow to replace this one.

**Comment by Niloth-p** [comment]
@zulipbot add "bug"

**Comment by neiljp** [comment]
@Niloth-p Thanks for filing this, though I believe this is essentially a DM-specific case of #607, both of which really apply to *any* empty narrow? For example, no starred messages, no mentions, and an empty search is likely the easiest to replicate.

I'm not sure how this is related to #1343 ?

Exiting from this situation is certainly a problem in general, though the `a` hotkey case is possibly a separate bug where it only triggers in certain UI locations. It'd certainly be useful to track down if that is the case.

**Comment by Niloth-p** [comment]
Thank you for the good suggestions on the replication part, I failed to consider an empty search.
Since these points weren't reported there, I wasn't sure if it was the same.

I didn't realise that the 'a' hotkey is a separate bug.
It's currently set to trigger only when focused on a MessageBox. So, I thought that was associated with the narrow being empty.
I've looked into adding support for the 'a' hotkey throughout the window.
Should I edit and re-frame this issue to the 'all messages' command?
